accessibility-checker-engine/help/landmark_name_unique.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "landmark_name_unique: Accessibility Checker Rule Help"
+---
+import "../../../styles/ToolHelp.scss"
+import { CodeSnippet, Tag } from "carbon-components-react";
+
+<div className="toolHelp">
+<Row>
+<Column colLg={16} colMd={8} colSm={4} className="toolHead">
+
+### Multiple landmark roles with the same `role` attribute are not disambiguated with a unique label or different parent  
+
+<div id="locLevel"></div>
+
+Multiple landmarks with the same `role` should have a unique `aria-labelledby` or `aria-label` or be nested in different parents
```

Comment:
@philljenkins 
> We can add that we're talking about "different parent regions" or "... different parent landarks", would that help you understand what exactly we mean?

"Different parent landmarks" is clearer, but I'm not sure that it's a sufficient technique for differentiating multiple landmarks with the same role? It might be ok for navigating with the virtual cursor, as long as the user remembers which landmark they are currently in. But in the landmark lists, while NVDA and JAWS do use a tree structure for showing the list of landmarks, the VoiceOver rotor doesn't. Screenshots below, all using SteveF's old test page: https://rawgit.com/stevefaulkner/HTML5accessibility/master/tests/structural-elements.html

NVDA INS+F7
![image](https://user-images.githubusercontent.com/3331913/145494480-de33cb72-f0bf-4af3-a0f7-0bcb0a8fe2c2.png)

JAWS INS+CTRL+R
![image](https://user-images.githubusercontent.com/3331913/145494511-83c3dbd7-6b61-4a5b-af69-854a904f47c9.png)

VoiceOver CTRL+ALT+U (photo taken with my phone because native Mac screenshot cmd+shift+3 and cmd+shift+5 don't work with the VO rotor open)
![image](https://user-images.githubusercontent.com/3331913/145494556-9837ee68-a578-43c0-a8d9-6daa1e5c0d49.png)

Reply:
maybe I'm not following this properly, bu the rotor provides an ability to jump to types of landmarks. When you go in via that way, you're not going to have the context of operation any more than if you used the R quick command to jump to the next region in JAWS.
I don't think that follows that hierarchy isn't a valid way to differentiate landmarks. Any way than a long page divided in sections, (think of a page on tools in a workshop) with a set of heading where the third level heading for each h2 is "Considerations". The user derives the context by their position in the hierarchy.
It's not AS disambiguated, but to say it's an error seems problematic to me.

Reply:
@mbgower 
> It's not AS disambiguated, but to say it's an error seems problematic to me.

Ok, I can buy that. I'll mark this conversation resolved.